pages/data.js

```diff
@@ -153,14 +160,24 @@ export default class Explore extends React.Component {
       const filterConfigs = datasets[activeDataset].filter_configs;
 
       // Setup our recordKeys
-      const recordKeys = Object.keys(data[activeDataset].records);
+      const { records } = data[activeDataset].compressed;
+      const recordKeys = Object.keys(records);
       const allUniqueRecords = {};
-      recordKeys.forEach(key => (allUniqueRecords[key] = [...new Set(data[activeDataset].records[key])]).sort());
+      recordKeys.forEach(key => (allUniqueRecords[key] = [...new Set(records[key])]).sort());
 
       // Filter our data, which will then be sent to Charts.js
-      const filteredData = filterData(data[activeDataset], filters);
+      const filteredData = filterData(data[activeDataset].compressed, filters);
```

Comment:
I think we can clean this a bit more by replacing `data[activeDataset].compressed` here with `records` which was defined above

Reply:
I dig it! https://github.com/texas-justice-initiative/website-nextjs/pull/180/commits/d636e80473a79d72be92d68a99e60b1671b321ec